src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURES_AMOUNT = 5;
+    public static final double MAX_FIGURE_SIDE_VALUE = 100;
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getDefaultFigure() {
+        return new Circle(10, Color.WHITE.toString().toLowerCase());
```

Comment:
Consider using `Color.WHITE.name().toLowerCase()` instead of `Color.WHITE.toString().toLowerCase()` to get the string representation of the enum constant. This follows the checklist recommendation to use `name()` for enum constants.